Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -21,7 +21,7 @@
 params = demisto.params()
 SPLUNK_TIME_FORMAT = "%Y-%m-%dT%H:%M:%S"
 VERIFY_CERTIFICATE = not bool(params.get('unsecure'))
-FETCH_LIMIT = int(params.get('fetch_limit', 50))
+FETCH_LIMIT = int(params.get('fetch_limit')) if params.get('fetch_limit') else 50
```

Comment:
This does not change the functionality

Reply:
@guykeller If the fetch_limit param is left empty in the instance configurations then an empty string is passed as the parameter. When doing params.get('fetch_limit', 50) where fetch_limit = "" the value is set to "" and not 50, because the default replacement to 50 would happen if it was None and not an empty string. On the other hand, checking if "" is False, hence causing FETCH_LIMIT to be 50.